core/domain/feedback_domain.py

```diff
@@ -53,15 +83,47 @@ def to_dict(self):
 
     @staticmethod
     def get_exp_id_from_full_thread_id(full_thread_id):
+        """Returns the full thread ID associated with this FeedbackThread 
+            instance.
```

Comment:
deindent by 4. But this docstring is wrong, I think? It doesn't concur with the method name, and it's a staticmethod so it's unrelated to this instance.